CMakeLists.txt

```diff
@@ -91,17 +97,30 @@ if(NOT UNIX AND IS_DIRECTORY ${DEPS_DIR})
    else ( )
      link_directories ( ${DEPS_DIR}/lib/Win32 )
    endif()
-   set( SDL_LIBRARY SDL )
+   set ( SDL_LIBRARY SDL )
+   if ( USE_SDL2 )
+     # explicitly added SDL2 as dependency since needed for runtime work
+     set ( SDL2_LIBRARY SDL2 )
+     message ( STATUS "Using SDL2 via SDL12 compatible layer" )
+   else ( )
+     set ( SDL2_LIBRARY "")
+   endif ()
    set ( SDLGFX_LIBRARY SDL_gfx )
    set ( SDLMIXER_LIBRARY SDL_mixer )
    set ( SDLIMAGE_LIBRARY SDL_image )
-   set ( PKG_DEPS_LDFLAGS ${SDLIMAGE_LIBRARY} ${SDLMIXER_LIBRARY} ${SDLGFX_LIBRARY} ${SDL_LIBRARY} ${OPENGL_LIBRARIES} )
+   set ( PKG_DEPS_LDFLAGS ${SDLIMAGE_LIBRARY} ${SDLMIXER_LIBRARY} ${SDLGFX_LIBRARY} ${SDL_LIBRARY} ${SDL2_LIBRARY} ${OPENGL_LIBRARIES} )
```

Comment:
Using this SDL2 added to the dependecies list which trigger copy of it to macos Frameworks library